script.js

```diff
@@ -34,9 +34,24 @@ function clearValidation() {
         })
 }
 
-function dataToForm(id) {
-    const value = $$("table").getItem(id);
-    $$("form").setValues(value);
+function randomInteger(min, max) {
+    let rand = min + Math.random() * (max + 1 - min);
+    return Math.floor(rand);
+}
+
+function addRandomUser() {
+    const countries = ["Germany", "USA", "Canada", "Russia", "China", "France", "Italy", "Spain"];
+    const names = ["John", "Bill", "Jacob", "William", "Liam", "Michael", "Alexander", "Noah"];
+    // const newUser = {
```

Comment:
please, don't commit unnecesary commented code

Reply:
It was accidentally, deleted.